packages/suite-web/package.json

```diff
@@ -33,5 +32,8 @@
         "jest": "^26.6.3",
         "rimraf": "^3.0.2",
         "typescript": "4.7.4"
+    },
+    "peerDependencies": {
+        "styled-components": "^5.3.6"
```

Comment:
why peer?

Reply:
Sorry, this commit should have been marked as WIP. What I am trying to solve is this warning is desktop:

It looks like there are several instances of 'styled-components' initialized in this application. This may cause dynamic styles to not render properly, errors during the rehydration process, a missing theme prop, and makes your application bigger without good reason.